content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+  },
+  members: [
+    {
+      _id: 0,
+      name: '10.192.3.58:27017',
+      health: 1,
+      state: 1,
+      stateStr: 'PRIMARY',
+      uptime: 4767,
+      optime: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T14:08:00.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastDurableWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      syncSourceHost: '',
+      syncSourceId: -1,
+      infoMessage: '',
+      electionTime: Timestamp({ t: 1669297790, i: 2 }),
+      electionDate: ISODate("2022-11-24T13:49:50.000Z"),
+      configVersion: 6,
+      configTerm: 1,
+      self: true,
+      lastHeartbeatMessage: ''
+    },
+    {
+      _id: 1,
+      name: '10.192.3.59:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 311,
+      optime: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T14:08:00.000Z"),
+      optimeDurableDate: ISODate("2022-11-24T14:08:00.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastDurableWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastHeartbeat: ISODate("2022-11-24T14:08:05.090Z"),
+      lastHeartbeatRecv: ISODate("2022-11-24T14:08:05.094Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.192.3.58:27017',
+      syncSourceId: 0,
+      infoMessage: '',
+      configVersion: 6,
+      configTerm: 1
+    },
+    {
+      _id: 2,
+      name: '10.192.4.208:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 56,
+      optime: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669298880, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-11-24T14:08:00.000Z"),
+      optimeDurableDate: ISODate("2022-11-24T14:08:00.000Z"),
+      lastAppliedWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastDurableWallTime: ISODate("2022-11-24T14:08:00.869Z"),
+      lastHeartbeat: ISODate("2022-11-24T14:08:05.104Z"),
+      lastHeartbeatRecv: ISODate("2022-11-24T14:08:03.615Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.192.3.59:27017',
+      syncSourceId: 1,
+      infoMessage: '',
+      configVersion: 6,
+      configTerm: 1
+    }
+  ],
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669298880, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669298880, i: 1 })
+}
+rs0 [direct: primary] test>
+```
+
+## HPE GreenLake for Containers: Data Science with MongoDB deployed on Kubernetes cluster
+
+Below steps illustrate interactive Jupyter notebook adding records and querying MongoDB deployed on 'hpe' Kubernetes cluster.
```

Comment:
Recommend rephrasing to "The below steps illustrate an interactive Jupyter notebook adding records and querying MongoDb deployed on the "hpe" Kubernetes cluster."

Reply:
done